spec/masamune/transform/define_table.dimension_spec.rb

```diff
@@ -149,7 +156,8 @@
         CREATE TABLE IF NOT EXISTS user_dimension_ledger
         (
           id SERIAL PRIMARY KEY,
-          user_account_state_type_id INTEGER REFERENCES user_account_state_type(id) DEFAULT default_user_account_state_type_id(),
+          cluster_type_id INTEGER NOT NULL REFERENCES cluster_type(id) DEFAULT default_cluster_type_id(),
+          user_account_state_type_id INTEGER REFERENCES user_account_state_type(id),
```

Comment:
do not define `DEFAULT` value for reference columns in ledger so that the can be coalesced correctly.